CHANGELOG-3.4.md

```diff
@@ -1,5 +1,9 @@
 # Changelog Lizmap 3.4
 
+## 3.4.7 - Not released
```

Comment:
```suggestion
## Unreleased

### Added

```

We are following the format : https://keepachangelog.com/en/1.0.0/
So it's need to be `## Unreleased` until it's not released otherwise, the parsing will break.

The parsing will accept subsection such as `Added`, `Changed` etc

Reply:
Sorry, can you rebase because I have updated the release 3.4 branch last week, but I have fixed this one already.
I have updated master just now.